src/ParticularTemplates/NServiceBusEndpoint/.template.config/template.json

```diff
@@ -130,9 +130,9 @@
       ],
       "defaultValue": "ConsoleApp"
     },
-    "UsesMSSQL": {
+    "NeedsSqlClient": {
       "type": "computed",
-      "value": "(transport == \"SQL\" || persistence == \"MSSQL\")"
+      "value": "(transport != \"SQL\" && persistence == \"MSSQL\")"
```

Comment:
I don't understand transport _not_ SQL, why is that?

(Later): Oh, is it because SQL Transport includes SQL Client as a transitive dependency?

Reply:
Yeah, no reason to add the package when it's already being referenced from the transport package.